eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -88,12 +88,16 @@ const AttendanceCard: React.FC<Props> = ({
    */
   const filteredCustomers = useMemo(
     () =>
-      allCustomers.filter(
-        ({ category, id }) =>
-          categories.includes(category) &&
-          !customers.find(({ id: customerId }) => customerId === id)
-      ),
-    [customers]
+      allCustomers
+        .filter(
+          ({ category, id }) =>
+            categories.includes(category) &&
+            !attendedCustomers.find(({ id: customerId }) => customerId === id)
+        )
+        .sort(({ name: a1, surname: a2 }, { name: b1, surname: b2 }) =>
```

Comment:
There are a few problems with the comparison function passed to `sort`.

```javascript
({ name: a1, surname: a2 }, { name: b1, surname: b2 }) =>
    `${a1} ${a2}`.toLowerCase() < `${b1} ${b2}`.toLowerCase() ? -1 : 1
```

* we should sort by last name first, then by first name
* the function needs to allocate memory to create two new strings
* the two new strings need to be lowercased, possibly needing more memory allocated for the lowercase version

This is really not needed at all. I doubt the impact of this exceeds a millisecond, so it would not be a huge problem if this went in, but for the sake of beauty, let's make it better.

I think the algorythm expressed with this pseudocode would yield the same result without allocating new memory. It also provides faster exit from the function, so that if the first letter of the last name is different the first name is not read at all.

* If a2 > b2 return 1
* If a2 < b2 return -1
* // if we're still executing at this point, it means a2 === b2
* if a1 > b1 return 1
* if a1 < b1 return -1
* // If we're at this point it means both `a`s and `b`s are equal
* return 0

This method should also be used for the sorting in `eisbuk-admin/src/store/selectors/attendance.ts`